src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/InternalCaseCreationContactDetailsAppenderTest.java

```diff
@@ -52,7 +52,7 @@ public void setUp() {
         when(asylumCase.read(IS_ADMIN, YesOrNo.class)).thenReturn(Optional.of(YesOrNo.YES));
 
```

Comment:
There is no changes here, just some alignment. Can you please rollback this file locally and then push this commit?